jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/proposal/quickfix/RemoveStaticModifierQuickFix.java

```diff
@@ -3,17 +3,24 @@
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at
+<<<<<<< HEAD
 * http://www.eclipse.org/legal/epl-2.0
+=======
+* http://www.eclipse.org/legal/epl-2.0.
+>>>>>>> 571b6ec (Change name and path for PreDestroyAnnotationQuickfix2)
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
+<<<<<<< HEAD
 *     IBM Corporation, Himanshu Chotwani - initial API and implementation
+=======
+*     IBM Corporation - initial API and implementation
+>>>>>>> 571b6ec (Change name and path for PreDestroyAnnotationQuickfix2)
```

Comment:
Clean up the comments here, looks left over from a merge/rebase

Reply:
solved